src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/EETypeNode.cs

```diff
@@ -195,7 +200,10 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly)
             OutputFinalizerMethod(factory, ref objData);
             OutputOptionalFields(factory, ref objData);
             OutputNullableTypeParameter(factory, ref objData);
-            OutputGenericInstantiationDetails(factory, ref objData);
+            if (factory.Target.Abi == TargetAbi.CoreRT)
```

Comment:
+1

This looks like a bug. Project N was switched to using this scheme months ago.